pallets/aleph/src/lib.rs

```diff
@@ -84,6 +106,39 @@ pub mod pallet {
         pub(crate) fn update_authorities(authorities: &[T::AuthorityId]) {
             <Authorities<T>>::put(authorities);
         }
+
+        pub(crate) fn update_emergency_finalizer() {
```

Comment:
I believe that names`QueuedEmergencyFinalizer` and  `NextEmergencyFinalizer` instead of `FutureEmergencyFinalizer` and `NextFutureEmergencyFinalizer` respectively would be more in line with the names in pallet session.

Reply:
Done.